docs/reference/signer-config.md

```diff
@@ -161,14 +202,16 @@ Example:
 			password used to encrypt the private keys in the DB.  All new
 			private keys will be encrypted using this password, which
 			must also be provided as the environment variable
-			<code>NOTARY_SIGNER_&lt;DEFAULT_ALIAS_VALUE&gt;</code>.</td>
+			<code>NOTARY_SIGNER_&lt;DEFAULT_ALIAS_VALUE&gt;</code>.
+			Please see the <a href="#notary-signer-envvars">environment variable</a>
```

Comment:
This link needs to be changed to `#environment_variables_required_if_using_mysql`